### PR TITLE
 DS-854: Aria attributes

### PR DESCRIPTION
**Pull request by pauzim**
* add iconButton aria-label

* refactor back to top button

* refactor DatePicker

* refactor FeturedCard

* fix icon Button test screen-object

* refactor modal

* refactor search

* refactor sidebar

* refactor sidepanel

* fix stories

* fix tests and stories

* refactor stepper

* fix stories

* refactor steps for storybook

* refactor videoplayer

* fix BackToTopButton stories

* add FAB aria labels

* fix tests

* fix videoplayer tests

* change backToTopButton props name

* create defaultIcon variable

* refactor props

* refactor props

* separate buttonProps object in FAB main button

* Fix `VideoPlayer` with aria labels

---------

### CODE REVIEW

File: packages/design-system/src/components/DatePicker/DatePickerComponents.tsx [hunk 1]
```diff
@@ -82,7 +89,11 @@ export const DatePickerComponents = {
             icon={<ChevronDownSvg className="-rotate-90" />}
             onClick={isYearPicker ? increaseYear : increaseMonth}
             className={getButtonClasses()}
-            aria-label={isYearPicker ? "increase year" : "increase month"}
```
Comment (zuzannanap) [inline packages/design-system/src/components/DatePicker/DatePickerComponents.tsx:85]: actually now i see that "next month"/"previous month" would be better :D

Comment (zuzannanap) [reply packages/design-system/src/components/DatePicker/DatePickerComponents.tsx:85]: oh sorry, i commented in the wrong place

**Comment by zuzannanap** [comment]
sth is wrong with modal - there is no close icon

File: packages/design-system/src/components/Sidebar/index.tsx [hunk 2]
```diff
@@ -147,7 +158,7 @@ function ToggleButton({ renderButton = true, className }: CollapseButtonProps) {
         size="sm"
         className="!border-sidebar-base-toggleButton !bg-sidebar-base-toggleButton"
         onClick={toggleCollapse}
-        aria-label={isCollapsed ? "Expand sidebar" : "Collapse sidebar"}
+        aria-label={isCollapsed ? notCollapsedAriaLabel : collapsedAriaLabel}
```
Comment (zuzannanap) [inline packages/design-system/src/components/Sidebar/index.tsx:161]: here it's the other way around

**Comment by zuzannanap** [comment]
videoplayer in storybook doesn't have aria-labels

**Comment by pauzim** [comment]
> videoplayer in storybook doesn't have aria-labels

dunno if you meant these ones? 
image

File: packages/design-system/src/components/DatePicker/DatePickerComponents.tsx [hunk 3]
```diff
@@ -16,13 +16,18 @@ import classNames from "classnames";
...
 export const DatePickerComponents = {
   Header: ({
     increaseMonth,
     decreaseMonth,
     increaseYear,
     decreaseYear,
+    increaseYearButtonProps = defaultDatePickerButtonProps.increaseYear,
+    decreaseYearButtonProps = defaultDatePickerButtonProps.decreaseYear,
+    increaseMonthButtonProps = defaultDatePickerButtonProps.increaseMonth,
+    decreaseMonthButtonProps = defaultDatePickerButtonProps.decreaseMonth,
```
Comment (monikaracka) [inline packages/design-system/src/components/DatePicker/DatePickerComponents.tsx:30]: I'm not sure it's worth the work, but I think the name previous/next would be better also for variables, props, etc.

File: packages/design-system/src/components/InputNumber/InputNumber.interface.ts [hunk 4]
```diff
@@ -18,7 +24,6 @@ export enum EActionType {
 export interface CounterButtonProps
   extends ClassName,
     ButtonHTMLAttributes<HTMLButtonElement> {
-  onClick: () => void;
   iconClassName?: string;
   disabled?: boolean;
```
Comment (monikaracka) [inline packages/design-system/src/components/InputNumber/InputNumber.interface.ts:28]: ButtonHTMLAttributes<HTMLButtonElement> should contain `disabled` prop.

**Review by monikaracka — APPROVED** [review]

**Pull request merged**